src/sourmash/__init__.py

```diff
@@ -1,12 +1,34 @@
-#! /usr/bin/env python
-"""
-An implementation of a MinHash bottom sketch, applied to k-mers in DNA.
+"""A library for computing hash sketches from DNA sequences, comparing
+them to each other, and plotting the results.
+
+Public API:
+
+    load_file_as_signatures(...) - load `[SourmashSignature, ]` from filename
+    load_file_as_index(...) - load collections of `SourmashSignature`s
+    save_signatures(...) - save `[SourmashSignature, ]`
+
+    class SourmashSignature - one or more hash sketches
+    class MinHash - hash sketch class
+
+Please see https://sourmash.readthedocs.io/en/latest/api.html for API docs.
+
+The sourmash code is available at github.com/dib-lab/sourmash/ under the
+BSD 3-Clause license.
 """
-import re
-import math
-import os
 from deprecation import deprecated
 
+__all__ = ['MinHash', 'SourmashSignature',
+           'load_one_signature',
```

Comment:
is `load_one_signature` also deprecated / to be removed in 5.0?

Reply:
no - it's used too widely/is too useful - see https://github.com/dib-lab/sourmash/issues/1062 for discussion.

I know it's a little inconsistent but I went for practicality here ;).

ref https://github.com/dib-lab/sourmash/pull/1279